imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -110,7 +119,8 @@ class CaseExplorer extends Component {
                     {isExpanded && (
                       <ul className='list bg-light-gray ma0 pl0 shadow-in-top-1'>
                         <CaseList
-                          allCases={hasCases}
+                          assignedToMe={assignedToMe}
```

Comment:
I can't see why `assignedToMe` is provided to CaseList

Reply:
Thank you!  I will remove this.